spec/books/dummy/actual_output.xhtml

```diff
@@ -0,0 +1,5 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <body>
+        <div>Hello, world!</div>
+    </body>
+</html>
```

Comment:
See other comments.  Let's bake to a temp file that gets cleaned up automatically, then we can gitignore these outputs.